autoload/sonokai.vim

```diff
@@ -183,6 +184,10 @@ function! sonokai#get_palette(style) "{{{
           \ 'none':       ['NONE',      'NONE']
           \ }
   endif
+  if a:transparent_background
```

Comment:
Is this too radical?

The original design is that it only effects the status line. If we directly change `bg1`, many hi groups will be effected.

Reply:
I missed this, will rollback this change soon

Reply:
This MR was closed automatically on force-push and I can't find a way to reopen it, so fixed version in https://github.com/sainnhe/sonokai/pull/58